packages/collector/test/tracing/database/mysql/app.js

```diff
@@ -14,6 +14,9 @@ process.on('SIGTERM', () => {
 });
 
 const agentPort = process.env.INSTANA_AGENT_PORT;
+if (process.env.MYSQL2_VERSION) {
```

Comment:
issue: MYSQL2_VERSION is set to "null" via the test. We mock it with "mysql2-null".

Reply:
updated